composer.json

```diff
@@ -34,22 +34,39 @@
         "twig/twig" : "2.7.4",
         "vlucas/phpdotenv" : "3.3.3",
         "yubico/u2flib-server" : "1.0.2",
-        "zendframework/zendframework" : "2.5.3",
+        "zendframework/zend-db" : "2.10.0",
+        "zendframework/zend-mvc" : "3.1.1",
+        "zendframework/zend-loader" : "2.6.0",
+        "zendframework/zend-inputfilter" : "2.10.0",
+        "zendframework/zend-soap" : "2.7.0",
+        "zendframework/zend-servicemanager" : "3.4.0",
+        "zendframework/zend-eventmanager" : "3.2.1",
+        "zendframework/zend-stdlib" : "3.2.1",
+        "zendframework/zend-escaper" : "2.6.0",
+        "zendframework/zend-config" : "3.1.0",
+        "zendframework/zend-filter" : "2.9.1",
+        "zendframework/zend-form" : "2.14.1",
+        "zendframework/zend-xmlrpc" : "2.7.0",
+        "zendframework/zend-session" : "2.8.5",
+        "zendframework/zend-json" : "3.1.0",
         "particle/validator": "2.3.4",
         "guzzlehttp/guzzle": "6.3.3",
         "kamermans/guzzle-oauth2-subscriber": "1.0.6",
         "league/omnipay": "3.0.2",
         "academe/omnipay-authorizenetapi": "3.1.1",
         "omnipay/stripe": "3.0.1",
         "waryway/php-traits-library": "1.0.4",
-        "paragonie/multi-factor": "0.2.2"
+        "paragonie/multi-factor": "0.2.2",
+        "zendframework/zend-mvc-i18n": "1.1.0",
+        "zendframework/zend-json-server": "3.1.0"
```

Comment:
Out of curiosity, how did you know which versions of these zendframework modules to use (appear unlike symfoni where versions are all the same)? Guessing are just using the most recent version of each.

Reply:
So I originally went off the composer.lock file and grabbed the locked versions of the original zend-framework components.  I then went through and upgraded them one by one to the most recent versions.  Zend says the components all have their own release cycle which I agree is confusing.  I like the symfony approach where the components try to match the same release as the other systems.